backend/routes/forms/new.py

```diff
@@ -26,5 +26,10 @@ async def post(self, request: Request) -> JSONResponse:
         except ValidationError as e:
             return JSONResponse(e.errors())
 
+        if await request.state.db.forms.find_one({"_id": form.id}):
+            return JSONResponse({
+                "error": "Form with same ID already exists."
+            })
```

Comment:
Let's return some status codes
```suggestion
            return JSONResponse({
                "error": "Form with same ID already exists."
            }, status_code=400)
```